packages/experiments/src/components/Button/ButtonVariants.types.ts

```diff
@@ -1,6 +1,8 @@
 import { IButtonProps } from './Button.types';
+import { IIconProps } from 'office-ui-fabric-react';
 
 export interface IButtonVariantProps extends IButtonProps {
+  iconProps?: IIconProps;
```

Comment:
I think we should consider marking this prop as deprecated in master and then remove it for Fabric 7 release, which would mean we wouldn't need to add it here.

Reply:
But how can we mark it as deprecated in the current `Button` in oufr if we still don't offer an alternative in that production component?

Reply:
By adding a `content` prop.. should be relatively easy. Unless we don't think that's a good permanent name for Button content, in which case we have a larger problem to address.

Reply:
And then we do the same for `iconProps`? Deprecate it in favor of a new `icon` prop?

Reply:
Possibly. It's either that or we carry over duplicate names for props even in Fabric 7 which seems cumbersome to take on for all of 7.x. I wonder what @dzearing thinks.